generators/server/files.js

```diff
@@ -845,16 +855,6 @@ const serverFiles = {
                 },
             ],
         },
-        {
-            condition: generator => !generator.reactive && generator.authenticationType === 'oauth2',
```

Comment:
This one was redundant with https://github.com/jhipster/generator-jhipster/blob/9ce17c10f78c8d36a0d034280b1bde43f86d44d1/generators/server/files.js#L450, which means that for a non reactive oauth2 application, the file was written twice.